01_draw_functions/01_subfunctions/MpptPlots.m

```diff
@@ -0,0 +1,59 @@
+function MpptPlots(sysvector, topics)
+% DisplayS MPPT plots
+
+fig1 = figure();
+fig1.Name = 'MPPT data';
+
+raw(1)=subplot(4,1,1);
+    if topics.sensor_mppt.logged
+        hold on;
+        plot(sysvector('sensor_mppt_0.mppt_amp_0').Time, sysvector('sensor_mppt_0.mppt_amp_0').Data,'LineWidth',2);
+        plot(sysvector('sensor_mppt_0.mppt_amp_1').Time, sysvector('sensor_mppt_0.mppt_amp_1').Data,'LineWidth',2);
+        plot(sysvector('sensor_mppt_0.mppt_amp_2').Time, sysvector('sensor_mppt_0.mppt_amp_2').Data,'LineWidth',2);
+        hold off
+    end
+title('Current [A]');
+ylabel('current [A]')
+legend('MPPT1','MPPT2','MPPT3');
+grid on;
+
+raw(2)=subplot(4,1,2);
```

Comment:
You get more flexibility if (after initializing with `raw(1)=...`), for all other assignments you just use `raw(end+1)=`. But no need to change this here, just maybe helpful in the future.